### PR TITLE
Italian Translation

### PR DESCRIPTION
**Pull request by richkmeli**
i've done the italian translation

### CODE REVIEW

**Comment by larsgrefer** [comment]
- [ ] Can you link a second italian-speaking person which can approve your translations?
- [ ] Please squash the commits into one

**Comment by SecUpwN** [comment]
>Can you link a second italian-speaking person which can approve your translations?

@larsgrefer, not sure if asking this will make translators find another person checking it.

>Please squash the commits into one

I guess @richkmeli does not even know how to, right? Why didn't you use [our weblate](https://hosted.weblate.org/accounts/login/github/?next=/projects/aimsicd/strings/), @richkmeli?

**Comment by richkmeli** [comment]
hi, i didn't use weblate because i saw the possibility to use it after than  i've made the job.

**Comment by SecUpwN** [comment]
>hi, i didn't use weblate because i saw the possibility to use it after than i've made the job.

I see, that makes sense. Nevertheless, would you please [squash all your commits into one](https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) so that we don't have a huge changeset when merging this pull request? If you're not able to, I'm afraid I'll have to neglect your PR since you also modified the `gradle` files and changed them back. Thanks for respecting.

File: AIMSICD/src/main/res/values-it/translatable_strings.xml [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+    <!-- Advanced user menu -->
+    <string name="deleted">Cancellato</string>
+    <string name="failed_to_delete">Fallito la rimozione</string>
+    <string name="double_quote_will_cause_db_error">"Stringa non aggiunta\n \" doppia quota causa errori al database "</string>
+    <string name="the_string_was_added_to_db">La stringa è stata aggiunta al Database</string>
+    <string name="failed_to_add_the_string_to_db">Fallito l\'aggiunta della stringa nel Database</string>
+    <string name="n_a">N/A</string>
+
+    <string-array name="pref_ui_icons_entries">
+        <item>@string/ui_icon_flat</item>
```
Comment (larsgrefer) [inline AIMSICD/src/main/res/values-it/translatable_strings.xml]: Is this really needed here?

Comment (SecUpwN) [inline AIMSICD/src/main/res/values-it/translatable_strings.xml]: What do you mean? Looks like @richkmeli translated all translatable strings.

Comment (richkmeli) [inline AIMSICD/src/main/res/values-it/translatable_strings.xml]: i didn't translate just all strings as an Translator, i also interpretate the context, therefore if you use an translator maybe will give you different words.
initialy i made the fork of AIMSICD for study the code and do some changes, and i took the oppotunity for traslate it.

**Comment by SecUpwN** [comment]
@richkmeli, I *very much* appreciate your enthusiasm about our app! But it seems as if your pull request here is causing too much trouble. Would you please do us the favor and do it the recommended way by pressing [HERE](https://hosted.weblate.org/accounts/login/github/?next=/projects/aimsicd/strings/) to ligin to weblate via GitHub, then press `Start new translation` and translate the strings that are provided? Please don't be mad at me when I close this pull request now. Thank you! :+1:

**Pull request closed without merging**